CHANGELOG.md

```diff
@@ -33,32 +33,35 @@ blog:
 - Fix the generator crashing for services containing function modules without a return type.
 Now the generator will handle these cases and use  `Promise<void>` as return type.
 
-
 # 1.18.0
 
-release-date:
-docs:
-blog:
-
-## knownIssues
-
--
+release-date: March 12, 2020
+docs: https://help.sap.com/doc/69202ef7e0a64767833782132648b855/1.0/en-US/index.html
+blog: https://blogs.sap.com/?p=1060707
 
 ## compatibilityNotes
 
--
-
-## newFunctionality
-
--
+- Some packages of the SAP Cloud SDK for JavaScript have been migrated to [external GitHub](https://github.com/SAP/cloud-sdk) and are now available as open-source software.
+As a result, the packages have been renamed as shown in the list below. 
+No Breaking changes are made. 
+The old versions of the packages will not receive further updates, so we heavily encourage switching to the new version of the packages.
+  - `@sap/cloud-sdk-util` was renamed to `@sap-cloud-sdk/util`
+  - `@sap/cloud-sdk-analytics` was renamed to `@sap-cloud-sdk/analytics`
+  - `@sap/cloud-sdk-core` was renamed to `@sap-cloud-sdk/core`
+  - `@sap/cloud-sdk-generator` was renamed to `@sap-cloud-sdk/generator`
+  - `@sap/cloud-sdk-test-util` was renamed to `@sap-cloud-sdk/test-util`
+
+- How to update your project
+1. Search for all your `dependencies`/`devDependencies`/`peerDependencies` in your `package.json`.
+1. Replace the old package name e.g., `@sap/cloud-sdk-core` with the new one, `@sap-cloud-sdk/core`.
+1. Use the stable version of the open source version e.g., `^1.18.0`.
+1. Deleting your `node_modules` and the `package-lock.json`.
+1. Install your dependencies again to reflect the changes via e.g., `npm i`.
+1. Search for your source code that uses the old packages as import and replace it with new names.
```

Comment:
```
### How to update your project
1. Search for all your `dependencies`/`devDependencies`/`peerDependencies` in your `package.json`.
1. Replace the old package name e.g., `@sap/cloud-sdk-core` with the new one, `@sap-cloud-sdk/core`.
1. Use the stable version of the open source version e.g., `^1.18.0`.
1. Deleting your `node_modules` and the `package-lock.json`.
1. Install your dependencies again to reflect the changes via e.g., `npm i`.
1. Search for your source code that uses the old packages as import and replace it with new names.
```

will render a bit nicer:

### How to update your project
1. Search for all your `dependencies`/`devDependencies`/`peerDependencies` in your `package.json`.
1. Replace the old package name e.g., `@sap/cloud-sdk-core` with the new one, `@sap-cloud-sdk/core`.
1. Use the stable version of the open source version e.g., `^1.18.0`.
1. Deleting your `node_modules` and the `package-lock.json`.
1. Install your dependencies again to reflect the changes via e.g., `npm i`.
1. Search for your source code that uses the old packages as import and replace it with new names.